docs/tutorials/storage.rst

```diff
@@ -0,0 +1,202 @@
+
+.. figure:: custom_utils.png
+    :align: center
+
+    Custom Utils
+
+After you click on Enter, a text editor will open. Copy and paste the following content:
+
+.. code-block:: python
+
+    from astropy.io import fits
+
+    class FitsFile():
+
+        def __init__(self, file):
+            self.name = file.name
+            self.file = file
+
+        def load(self):
+            return fits.open(self.file)
+
+        def save(self, outfile):
+            os.system('ln -s ' + self.name + ' ' + outfile.name)
+
+    def load(infile):
+        fits_file = FitsFile(infile)
+        return fits_file.load()
+
+    def save(fits_file, out_file):
+        fits_file.save(out_file)
+
+Figure below shows the text editor with the content:
+
+.. figure:: fits_serializer.png
+    :align: center
+
+    FITS Serializer
+
+Now, we are ready to import files as described in :ref:`previous section <Loading a pickled file>`, but this time specifying the `Data Type` as ``utils.fits_file.FitsFile``. This is shown in the figure below (For one FITS File):
+
+.. figure:: custom_serialized_artifact_upload.png
+    :align: center
+
+    Uploading Artifact with a Custom Serializer
+
+Now we are ready to use the files. Follow similar steps for all the FITS files you want to use. In our case, we want to visualize them using ``make_lupton_rgb`` function in ``astropy.visualization`` package. The pipeline `Visualize FITS` is shown below:
+
+.. figure:: visualize_fits.png
+    :align: center
+
+    Pipeline Visualize FITS
+
+The `pipeline` has a single operation called `VisualizeFITS` that uses the FITS file we uploaded with custom serializer.
+
+Using data available different storage backends
+-----------------------------------------------
+In DeepForge there are several options to upload/import an artifact to/from.
+
+In this example, we will look into importing an existing file in `sciserver-files <https://apps.sciserver.org/dashboard/files/uservolumes>`_ for use in DeepForge. If you have a file called `dataset.npz` in sciserver-files, in a user volume called `Datasets` in the `Storage Volume Pool` and you want to Import it in DeepForge, Select `Import existing data` from the Artifact Index as highlighted below:
+
+.. figure:: import_existing_data.png
+    :align: center
+
+    Importing Existing Data
+
+After you click on Import Existing Data, a Modal as shown below will pop up, select `Sciserver Files Service` from the dropdown menu. Provide your `username` and `password` for SciServer and select correct volume pool and file path as shown in the figure below:
```

Comment:
- "Import Existing Data" should be either quoted or italicized since you are referring to it as a noun and not a verb phrase.
- "Modal" should not be capitalized.
- "select *the* correct volume pool"